### DISCUSSION TITLE
Clone: 𝙂𝙞𝙫𝙚 𝙩𝙝𝙚 𝙨𝙤𝙪𝙧𝙘𝙚 𝙘𝙝𝙖𝙩 𝙞𝙣𝙛𝙤: 𝑌𝑜𝑢 ℎ𝑎𝑣𝑒 30𝑆𝑒𝑐 𝑡𝑜 𝑑𝑜 𝑡ℎ𝑖𝑠..  𝙄𝙣𝙫𝙖𝙡𝙞𝙙 𝙧𝙚𝙥𝙡𝙖𝙮 𝙢𝙚𝙨𝙨𝙖𝙜𝙚 ❗️

### DISCUSSION THREAD

**Discussion by entrysky** [m4mallu/clonebot#111] [Q&A] [answered]
Dear Rajan,

i followed the instructions, i installed on linux VPS, bot worked.
 But when I type Source: the bot responds with the following error: 
Clone:
𝙂𝙞𝙫𝙚 𝙩𝙝𝙚 𝙨𝙤𝙪𝙧𝙘𝙚 𝙘𝙝𝙖𝙩 𝙞𝙣𝙛𝙤:
𝑌𝑜𝑢 ℎ𝑎𝑣𝑒 30𝑆𝑒𝑐 𝑡𝑜 𝑑𝑜 𝑡ℎ𝑖𝑠..
𝙄𝙣𝙫𝙖𝙡𝙞𝙙 𝙧𝙚𝙥𝙡𝙖𝙮 𝙢𝙚𝙨𝙨𝙖𝙜𝙚 ❗️
![Screenshot_2022-10-27-08-57-03-60_cdcac83adb6dcd89b46a8d7c61e26532.jpg](https://user-images.githubusercontent.com/65966878/198173014-a4f37a16-012c-428d-8ce9-e9e41e4cbd20.jpg)
Anyone have the same error as me?Please guide me. Thank

**Comment by m4mallu** [comment]
1. Clear bot history
2. Start the bot
3. Tap the `Source` Button
4. Input the `Source Info` within 30Sec

Clearing history may resolve this issue.

**Reply by entrysky** [reply]
Hi I follow your instructions. It worked.
But now when I choose Target. Again received the following error message:

A̶c̶c̶e̶s̶s̶ ̶D̶e̶n̶i̶e̶d̶

𝘜𝘴𝘦𝘳 𝘪𝘴 𝘯𝘰𝘵 𝘢𝘯 𝘢𝘥𝘮𝘪𝘯 𝘰𝘳 𝘥𝘰𝘦𝘴𝘯'𝘵 𝘩𝘢𝘷𝘦
𝘱𝘰𝘴𝘵𝘪𝘯𝘨 𝘱𝘳𝘪𝘷𝘪𝘭𝘢𝘨𝘦𝘴 𝘪𝘯 𝘵𝘩𝘦 𝘨𝘪𝘷𝘦𝘯 𝘤𝘩𝘢𝘵
![Screenshot_2022-10-27-13-12-01-06_cdcac83adb6dcd89b46a8d7c61e26532.jpg](https://user-images.githubusercontent.com/65966878/198205483-34c92529-b5ea-4b91-a85e-e2ee3812db09.jpg)
Although, I have added the Bot to the Group and Granted Admin Permissions.

What action am I missing?

Thank you very much!

**Comment by m4mallu** [comment]
First, You have only 30 sec to react/input that reply message. The messages sent after the specified time will be considered as a `custom caption text`. So be replied within the time. The `Session user account holder` should have an `admin` with `posting privileges` in the target chat id given in the bot. Be ensure the above thing before doing the same.

**Reply by entrysky** [reply]
Yes, I make sure that the bot has given Admin rights, User also has Admin rights![Screenshot_2022-10-27-18-04-46-99_cdcac83adb6dcd89b46a8d7c61e26532.jpg](https://user-images.githubusercontent.com/65966878/198268465-347257a0-6fa4-4ca0-94a0-f9f0ba17c586.jpg)

When I try to enter Target I get this message, ![Screenshot_2022-10-27-13-12-01-06_cdcac83adb6dcd89b46a8d7c61e26532.jpg](https://user-images.githubusercontent.com/65966878/198268582-1f3b2ab1-69dd-4086-a45c-54eebc34e1f0.jpg)

**Comment by m4mallu** [comment, marked as answer]
Seems to be an edited repo you are using, and won't get any assistance here. 
I didn't coded such things in this repo like `Message sent. we'll reply as soon as possible`.

Else, Use updated repo  (main) and make sure you are doing things in a right way. For more about the configuration and working, go through the documentation provided. The repo currently don't have such issues.
